### DISCUSSION TITLE
How to manage my code changes and your updates

### DISCUSSION THREAD

**Discussion by dugufly** [google/filament#5660] [Q&A]
I want to modify the filament engine code, but your code will be updated regularly. How can I keep my changes and your updates? The method of code merging is troublesome.

**Comment by bejado** [comment]
What types of modifications to the engine are you trying to make?

**Reply by dugufly** [reply]
I want to modify IBL_INTEGRATION and other shader src code。 
The code location at ：
\shaders\src
\filament\src

**Reply by bejado** [reply]
We make changes to code under those paths quite frequently, I'm not sure there's any way to avoid merge conflicts.
